stress/stress_test.go

```diff
@@ -50,12 +60,17 @@ func TestTimer_StopTimer(t *testing.T) {
 }
 
 func TestTimer_Elapsed(t *testing.T) {
+	const delay = 2 * time.Second
 	tmr := NewTimer()
 	time.Sleep(2 * time.Second)
 	tmr.StopTimer()
 	e := tmr.Elapsed()
-	if time.Duration(2*time.Second) > e || e > time.Duration(3*time.Second) {
-		t.Errorf("expected around %s got %s", time.Duration(2*time.Second), e)
+	delayadj := delay
+	if runtime.GOOS == "windows" {
+		delayadj -= windowsInaccuracy
+	}
+	if e < delayadj || e > delay+time.Second {
```

Comment:
Maybe I should make a change to this test to skip it on Windows for now. I'll keep this PR open, but at least we can stop having random test failures.